crates/aptos/src/ascii/mod.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::common::types::CliResult;
+use clap::Parser;
+use indoc::indoc;
+
+/// Aptos Ascii Art
+///
+/// This tool is used to celebrate aptos in spectacular ascii
+/// fashion!
+#[derive(Debug, Parser)]
+pub struct AsciiTool {}
+
+impl AsciiTool {
+    pub async fn execute(self) -> CliResult {
```

Comment:
This tool doesn't implement the required interfaces to be the same as other commands in the tool.

Reply:
What is the required interface?

This has the async execute and there was no explicit interface I could find?